src/components/password/_password.scss

```diff
@@ -0,0 +1,89 @@
+/* ===============Text Input================== */
+.cmp-adaptiveform-password {
+    color: $dark-gray;
+    font-size: $font-md;
+    margin: $field-margin;
+    display: flex;
+    flex-direction: column;
+  }
+  .cmp-adaptiveform-password__label-container{
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+  }
+  .cmp-adaptiveform-password__widget {
+    margin-top: $label-top-margin;
+    height: $field-height;
+    border-radius: 0.25rem;
+    border: 1px solid $dark-gray;
+    font-size: $font-md;
+    color: $dark-gray;
+    padding: 0 $field-padding-left;
+    position:relative;
+  }
+  .cmp-adaptiveform-password__longdescription {
+    color: $light-gray;
+    background-color: $very-light-gray;
+    font-size: $font-sm;
+    margin-top: $error-top-margin;
+    margin-bottom: 5px;
+    padding: 10px;
+    p{
+      margin: 0;
+      padding: 0;
+    }
+  }
+  .cmp-adaptiveform-password__shortdescription {
+    font-size: $font-sm;
+    margin-top: $error-top-margin;
+  }
+  .cmp-adaptiveform-password__questionmark{
+    display:inline-block;
+    width:1rem;
+    height:1rem;
+    border-radius: 9px;
+    background: url(./resources/images/question.svg) center center / cover no-repeat,#969696;
+    cursor : pointer;
+    // below properties for backward compatibility
+    position: absolute;
+    right: 20px;
+  }
+  // below properties for backward compatibility
+  .cmp-adaptiveform-password__label-container .cmp-adaptiveform-password__questionmark{
+    position: unset;
+    right: unset;
+  }
+  .cmp-adaptiveform-password__errormessage{
+    color: $error;
+    font-size: $font-sm;
+    margin-top: $error-top-margin;
+  }
+  .cmp-adaptiveform-password[data-cmp-valid=false]{
+    background: $light-error;
+    border: 0 solid $error;
+    border-inline-start-width: 4px;
+    padding-inline-start: 5px;
+  }
+  .cmp-adaptiveform-password[data-cmp-valid=true]{
+    border: 0 solid $success;
+    border-inline-start-width: 4px;
+    padding-inline-start: 5px;
+  }
+  .cmp-adaptiveform-password__input-wrapper{
+    position:relative;
+  }
+  .cmp-adaptiveform-password__input-wrapper .cmp-adaptiveform-password__widget{
+    width:100%
+  }
+  .cmp-adaptiveform-password__input-wrapper .cmp-adaptiveform-password__eyeicon{
+    display:inline-block;
+    position: absolute;
+    right: -5px;
```

Comment:
This is not RTL complaint. As you have fixed the direction. So you also need to add something like:
[dir="rtl"] .cmp-adaptiveform-password__input-wrapper .cmp-adaptiveform-password__eyeicon {
  left: -5px;
  right: unset;
}

Also why it is -5px?

Reply:
fixed, removed -5px.